src/Sylius/Bundle/ApiBundle/Resources/config/api_platform/resources/Customer.xml

```diff
@@ -28,10 +28,104 @@
         </validationContext>
 
         <operations>
+
+            <!-- Admin operations-->
+
+            <operation class="ApiPlatform\Metadata\GetCollection" uriTemplate="/admin/customers">
+                <normalizationContext>
+                    <values>
+                        <value name="groups">
+                            <values>
+                                <value>sylius:admin:customer:index</value>
+                            </values>
+                        </value>
+                    </values>
+                </normalizationContext>
+                <filters>
+                    <filter>sylius.api.customer_group_filter</filter>
+                </filters>
+            </operation>
+
+            <operation class="ApiPlatform\Metadata\Get" uriTemplate="/admin/customers/{id}">
+                <normalizationContext>
+                    <values>
+                        <value name="groups">
+                            <values>
+                                <value>sylius:admin:customer:show</value>
+                            </values>
+                        </value>
+                    </values>
+                </normalizationContext>
+            </operation>
+
             <operation
                 class="ApiPlatform\Metadata\Get"
-                uriTemplate="/admin/customers/{id}"
+                uriTemplate="/admin/customers/{id}/statistics"
+                controller="Sylius\Bundle\ApiBundle\Controller\GetCustomerStatisticsAction"
             >
+                <normalizationContext>
+                    <values>
+                        <value name="groups">
+                            <values>
+                                <value>sylius:admin:customer:statistics:show</value>
+                            </values>
+                        </value>
+                    </values>
+                </normalizationContext>
+            </operation>
+
+            <operation class="ApiPlatform\Metadata\Post" uriTemplate="/admin/customers" >
+                <validationContext>
+                    <values>
+                        <value name="groups">
+                            <values>
+                                <value>sylius</value>
```

Comment:
```suggestion
```
This group is already set at the top.

Reply:
I know, but without this group declared here also the validation tests are failing 🤔

Reply:
Perhaps they are no longer merging 🤔